resources/locale/en.yml

```diff
@@ -32,6 +32,12 @@ askvortsov-pwa:
         push_notif_preference_default_to_email_text: If true, users will have push notifications enabled by default for all notification types where email notifications are enabled by default. If false, users will have push notifications disabled by default.
         user_max_subscriptions_label: Max Push Subscriptions Per User
         user_max_subscriptions_text: How many push subscriptions should each user be able to have at a time? When a user receives a push notification, each subscription will result in an API call. Each subscription typically corresponds to a browser the user has used.
+        windowControlsOverlay_label: Enable Window Controls Overlay?
+        windowControlsOverlay_text: |
+          If true, users will able to enable Window Controls Overlay to create an immersive, native-like experience. Window Controls Overlay enables or not depends on user's preference and browser's compatibility. You also should be sure you've done something to match this feature.
```

Comment:
Sorry, it may not be described clearly. As the text mentioned, we must do something to adapt the UI changes.
Specifically speaking, after enabling this feature, the titlebar will disappear and only leave the three controls. Flarum doesn't do anything for that, so you'll have a bad window. For example:
![Not adapted with Window Controls Overlay](https://github.com/askvortsov1/flarum-pwa/assets/77269678/2754771f-c3b4-460d-975d-4e88f01b4fb8)
Just like that. So you're needed to do is just write some CSS to "match this feature". For example (Please pay attention to the position changes of the elements on the titlebar):
![Adapted](https://wvbimg2-1310561333.cos.ap-tokyo.myqcloud.com/2023-08-30/1693384524-592181-image.png)
That's all. Since the description text is considered as incomprehensible, maybe it's necessary to rewrite it......If someone can provide help, I would be grateful.